config/settings/test.yml

```diff
@@ -12,6 +12,7 @@ features:
     school_direct_tuition_fee: true
     pg_teaching_apprenticeship: true
   redirect_education_domain: true
+  funding: true
```

Comment:
We should probably make sure tests are passing with the flag false as that's how it is in production.

Reply:
thanks Graham - because the route creation itself is feature flagged, if I set `funding: false` in the test.yml and then try to switch it on in the feature test via `feature_funding: true`, the tests fail. This is because the route is never created.

```
     ActionController::RoutingError:
       No route matches [GET] "/funding/trainee-summary"
```

If we remove the routing constraint, testing becomes easier but the user could access the URL directly. What do you think we should do? John's PR also has `funding: true` in the test.yml (maybe because of the same issue.)

Reply:
Ah right. You can switch the feature flag in the test and reload the routes with `Rails.application.reload_routes!`. I'll approve as is but let me know if you want to have a look at how you would do that for the future.

Reply:
oh nice! Got it working :)